cvat/apps/engine/model_utils.py

```diff
@@ -18,3 +22,86 @@ class Undefined:
 The reverse side of one-to-one relationship.
 May be undefined in the object, should be accessed via getattr().
 """
+
+
+_ModelT = TypeVar("_ModelT", bound=models.Model)
+
+
+def bulk_create(
+    db_model: type[_ModelT],
+    objs: Iterable[_ModelT],
+    batch_size: int | None = ...,
+    ignore_conflicts: bool = ...,
+    update_conflicts: bool | None = ...,
+    update_fields: Sequence[str] | None = ...,
+    unique_fields: Sequence[str] | None = ...,
+    *,
```

Comment:
I would suggest to move this above `batch_size`.

Reply:
I used API from the [documentation](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#bulk-create) (and from the VS code django stub). Do you think the argument order should be changed?

`bulk_create(objs, batch_size=None, ignore_conflicts=False, update_conflicts=False, update_fields=None, unique_fields=None)`

Reply:
By "this", I meant the asterisk. Basically, I'm suggesting to make all arguments with defaults kw-only.

Reply:
We can make the parameters kw-only, but in the original function they are positional.